src/main/java/Duke.java

```diff
@@ -1,10 +1,191 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+/**
+ * This class is to build a personal assistance chat-bot called "Kaman"
+ * (Customised from Duke)
```

Comment:
The word "Customised" is a British spelling and violates the coding standard of using American spelling only.
Perhaps you may want to correct it to "Customized" instead.